src/flowchem/components/valves/valve.py

```diff
@@ -211,13 +211,6 @@ def _connect_positions(self,
                               "This can be due to exclusion of certain connections by setting positions_not_to_connect")
 
     async def get_position(self) -> list[list[int]]:
-        """
-        Get the current valve position.
-
-        Returns:
-            tuple: The current position of the valve.
```

Comment:
why remove return value?

Reply:
The docstring will be fixed in the Doc branch. It can avoid conflict problems when merging the Doc into the main_all.